spec/requests/api/v1/accounts_show_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'GET /api/v1/accounts/{account_id}' do
+  it 'returns account entity as 200 OK' do
+    account = Fabricate(:account)
+
+    get "/api/v1/accounts/#{account.id}"
+    response_body = body_as_json
+
+    aggregate_failures do
+      expect(response).to have_http_status(200)
+      expect(response_body[:id]).to eq(account.id.to_s)
+    end
+  end
+
+  it 'returns 404 if account not found' do
+    get '/api/v1/accounts/1'
+    response_body = body_as_json
+
+    aggregate_failures do
+      expect(response).to have_http_status(404)
+      expect(response_body[:error]).to eq('Record not found')
+    end
+  end
+
+  context 'when with token' do
+    it 'returns account entity as 200 OK if token is valid' do
+      account = Fabricate(:account)
+      user = Fabricate(:user, account: account)
+      token = Fabricate(:accessible_access_token, resource_owner_id: user.id, scopes: 'read:accounts').token
+
+      get "/api/v1/accounts/#{account.id}", headers: { Authorization: "Bearer #{token}" }
+      response_body = body_as_json
+
+      aggregate_failures do
+        expect(response).to have_http_status(200)
+        expect(response_body[:id]).to eq(account.id.to_s)
+      end
+    end
+
+    it 'returns 403 if scope of token is invalid' do
```

Comment:
This is fine for initial conversion, but once we get past a few of these we probably do want to preserve something like the shared example from the controller specs for the bad scope access scenario.

Reply:
Fixed.
In my initial draft, I asserted all key value pairs.
However, it became too long.
So, I reduced assertions to only id.
Assignment body_as_json to response_body local variable is a trace of it. 

I prefer to symbolize keys when asserting the response body.

Reply:
In request_spec, I like to keep things as simple and readable as possible.

I am not sure if I should test in all scopes.
If it is as an example, it is enough to just one.

If we want to cover all cases, I can write it like this.
But this is complicated.

```ruby
  context 'when with token' do
    valid_scopes = [
      'read',
      'read:accounts',
    ]
    invalid_scopes = Doorkeeper.config.scopes.to_a - valid_scopes

    valid_scopes.each do |valid_scope|
      it "returns account entity as 200 OK if scope of token is #{valid_scope}" do
        account = Fabricate(:account)
        user = Fabricate(:user, account: account)
        token = Fabricate(:accessible_access_token, resource_owner_id: user.id, scopes: valid_scope).token

        get "/api/v1/accounts/#{account.id}", headers: { Authorization: "Bearer #{token}" }

        aggregate_failures do
          expect(response).to have_http_status(200)
          expect(body_as_json[:id]).to eq(account.id.to_s)
        end
      end
    end

    invalid_scopes.each do |invalid_scope|
      it "returns 403 if scope of token is #{invalid_scope}" do
        account = Fabricate(:account)
        user = Fabricate(:user, account: account)
        token = Fabricate(:accessible_access_token, resource_owner_id: user.id, scopes: invalid_scope).token

        get "/api/v1/accounts/#{account.id}", headers: { Authorization: "Bearer #{token}" }

        aggregate_failures do
          expect(response).to have_http_status(403)
          expect(body_as_json[:error]).to eq('This action is outside the authorized scopes')
        end
      end
    end
  end
```